docs/operations/metrics.md

```diff
@@ -516,8 +516,12 @@ These metrics are available on operating systems with the cgroup kernel feature.
 |------|-----------|----------|------------|
 |`cgroup/cpu/shares`|Relative value of CPU time available to this process. Read from `cpu.shares`.||Varies|
 |`cgroup/cpu/cores_quota`|Number of cores available to this process. Derived from `cpu.cfs_quota_us`/`cpu.cfs_period_us`.||Varies. A value of -1 indicates there is no explicit quota set.|
+|`cgroup/cpuacct/usage`|Reports the total CPU time (in nanoseconds) consumed by all tasks in this process||Varies|
```

Comment:
```suggestion
|`cgroup/cpuacct/usage/ns`|Reports the total CPU time (in nanoseconds) consumed by all tasks in this process||Varies|
```

Include the unit to make it easier for consumers of the metric to understand the value without having to read the docs.